src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -145,22 +165,11 @@ private async Task<string> Fetch(string url, IReadOnlyDictionary<string, string>
             return await response.EnsureSuccessStatusCode().Content.ReadAsStringAsync();
         }
 
-        private async Task<HttpResponseMessage> SendRequest(HttpRequestMessage request)
-        {
-            using (PerfScope.Start($"[{nameof(OpsConfigAdapter)}] Executing request '{request.Method} {request.RequestUri}'"))
-            {
-                _credentialProvider?.Invoke(request);
-
-                await FillOpsToken(request.RequestUri.AbsoluteUri, request);
-
-                return await _http.SendAsync(request);
-            }
-        }
-
-        private async Task<string> FetchValidationRules(string url, string repositoryUrl = "", string branch = "", DocsEnvironment? environment = null)
+        private async Task<string> FetchValidationRules(string routePath, string repositoryUrl = "", string branch = "", DocsEnvironment? environment = null)
         {
             try
             {
+                var url = $"{BuildServiceEndpoint(environment)}/{routePath.TrimStart('/')}";
```

Comment:
we can assert the url path always starts with `/`
```suggestion
                var url = $"{BuildServiceEndpoint(environment)}{routePath}";
```

Reply:
this is a best practice heard from the service side,  it can still be right regardless of that the caller doesn't start with `/`